pkg/languages/languages.go

```diff
@@ -98,16 +95,17 @@ func (l *Languages) PopulateConfigs() {
 	}
 }
 
-func CreateLanguages(dest string) *Languages {
+func CreateLanguages(builders embed.FS, dest string) *Languages {
```

Comment:
Maybe let's rename this to `CreateLanguagesFromEmbedFS`

Reply:
sounds good